nipoppy_cli/docs/source/conf.py

```diff
@@ -4,6 +4,8 @@
 https://www.sphinx-doc.org/en/master/usage/configuration.html
 """
 
+from nipoppy._version import __version__
```

Comment:
@Remi-Gau , from my understanding of https://github.com/nipoppy/nipoppy/pull/308#discussion_r1701362261, the `nipoppy/_version.py` file is only create during the pip install.
How does it work for dev and tests, if the file doesn't exist locally? I'm probably just misunderstanding how the `_version.py` file works.

Reply:
well in all cases you need to install the package and therefore the `_version.py` will be created